updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc

```diff
@@ -0,0 +1,136 @@
+:_content-type: ASSEMBLY
+[id="updating-restricted-network-cluster-OSUS"]
+= Updating a restricted network cluster using the OpenShift Update Service
+include::_attributes/common-attributes.adoc[]
+:context: updating-restricted-network-cluster-OSUS
+
+toc::[]
+
+[id="update-restricted-network-cluster-update-service"]
+
+include::modules/update-service-overview.adoc[leveloffset=+1]
+
+.Additional resources
+
+* xref:../../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[Understanding upgrade channels and releases]
+
+For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs. However, clusters in a restricted network have no way to access public APIs for update information.
+
+To provide a similar update experience in a restricted network, you can install and configure the OpenShift Update Service locally so that it is available within a disconnected environment.
+
+The following sections describe how to provide over-the-air updates for your disconnected cluster and its underlying operating system.
+
+[id="update-service-prereqs"]
+== Prerequisites
+
+* Have access to the internet to obtain the necessary container images.
+* Have write access to a container registry in the restricted-network environment to push and pull images. The container registry must be compatible with Docker registry API v2.
+* You must have the `oc` command-line interface (CLI) tool installed.
+* For more information on installing Operators, see xref:../../operators/user/olm-installing-operators-in-namespace.adoc#olm-installing-operators-in-namespace[Installing Operators in your namespace].
+
+[id="registry-configuration-for-update-service"]
+== Configuring access to a secured registry for the OpenShift update service
+
+If the release images are contained in a secure registry, complete the steps in xref:../../registry/configuring-registry-operator.adoc#images-configuration-cas_configuring-registry-operator[Configuring additional trust stores for image registry access] along with following changes for the update service.
```

Comment:
The trust stores are for X.509 certificate authorities, to allow HTTPS and other TLS-based registry connections, etc.  So the cluster deciding whether it should trust the registry.  There are also pull-secrets and similar, for the registry to decide whether it should trust the cluster.  I think "a secure registry" could be tightened down to something like "a registry whose HTTPS X.509 certificate is signed by a custom certificate authority" or some such here (perhaps we have existing language for this in the trust-store docs?), so folks are more likely to click through when they need to, and less likely to click through when they don't need to.  But also, even if we don't ace the send-off wording, I expect readers will eventually figure things out, so no worries if you want to leave the current wording in place.

Reply:
I'll see if I can track down specific wording in the documentation, otherwise I'll try something like "a registry whose HTTPS X.509 certificate is signed by a custom certificate authority" and see if it's not too wordy